src/routes/components/components.json

```diff
@@ -2856,5 +2856,25 @@
 		"category": "Forms & User Input",
 		"tags": ["forms", "rich text editor", "audio", "components and libraries"],
 		"stars": 9
+	},
+	{
+		"title": "SvelteKit Cordova/Capacitor Adapter",
+		"url": "https://github.com/ptkdev/sveltekit-cordova-adapter",
+		"description": "Adapter for build mobile apps with Svelte Kit and Apache Cordova or Ionic Capacitor",
+		"npm": "@ptkdev/sveltekit-cordova-adapter",
+		"addedOn": "2023-02-11T00:00:00Z",
+		"category": "SvelteKit Adapters",
+		"stars": 3,
+		"tags": ["ssg", "mobile apps", "cordova", "capacitor", "ionic", "integrations"]
```

Comment:
The tag `ssg` is new.
- It's the adapter that is an SSG or the SvelteKit application ?
- Is it necessary ?
- Can it be added to already existing components ?

Reply:
I use a fork of svelte-kit static-adapter, "ssg" is more accurate.

Reply:
In that case, the tag should be added to (at least):
- `@sveltejs/adapter-static`
- `@macfja/svelte-adapter-neutralino`
- `sveltekit-adapter-html-like`
- `svelte-adapter-github`
- `svelte-adapter-static-digitalocean`

Reply:
Perfect, fixed! Thanks